docker/maistra-builder_2.4.Dockerfile

```diff
@@ -202,6 +203,13 @@ RUN mkdir /tmp/su-exec && cd /tmp/su-exec && \
 RUN curl -o /usr/bin/bazel -Ls https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-x86_64 && \
     chmod +x /usr/bin/bazel
 
+# FPM
+RUN git clone https://github.com/jordansissel/fpm /tmp/fpm && \
+    cd /tmp/fpm && \
+    git reset --hard ${FPM_VERSION} && \
+    make install && \
+    rm -rf /tmp/*
```

Comment:
Alternatively, you can try doing a shallow clone for that given commit if the repo is big.

```
mkdir -p /tmp/fpm && cd /tmp/fpm && git init && \                                                                                          ─╯
git remote add origin https://github.com/jordansissel/fpm && \
git fetch --depth 1 origin ${FPM_VERSION} && \
git checkout FETCH_HEAD
```

Reply:
Cool. We can submit this upstream. I blindly copied this snippet from them.

Reply:
Good idea. I will send the PR.